src/careamics/models/lvae/layers.py

```diff
@@ -178,17 +185,24 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs, gated=True)
 
 
-class GateLayer2d(nn.Module):
+class GateLayer(nn.Module):
     """
     Double the number of channels through a convolutional layer, then use
     half the channels as gate for the other half.
     """
 
-    def __init__(self, channels, kernel_size, nonlin=nn.LeakyReLU):
+    def __init__(
+        self,
+        channels: int,
+        conv_strides: tuple[int] = (2, 2),
```

Comment:
conv_dims

Reply:
We need to discuss this in more detail to figure out the way we handle 2D/3D